metadata-io/src/main/java/com/linkedin/metadata/graph/GraphFilters.java

```diff
@@ -0,0 +1,18 @@
+package com.linkedin.metadata.graph;
+
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+
+@Data
+@AllArgsConstructor
+public class GraphFilters {
```

Comment:
For other services (e.g. search service) we use PDLs to model the filter types...Any reason why we shouldn't keep that consistent here?

Reply:
These aren't exposed via API yet so ill keep them just in java for now so its easier to change. Once we've settled on them let's move them to pdl.